web/server/codechecker_server/api/report_server.py

```diff
@@ -1641,7 +1643,9 @@ def changeReviewStatus(self, report_id, status, message):
                 codechecker_api_shared.ttypes.ErrorCode.GENERAL, msg)
 
         with DBSession(self.__Session) as session:
-            res = self._setReviewStatus(session, report_id, status, message)
+            report = session.query(Report).get(report_id)
+            res = self._setReviewStatus(
+                session, report.bug_id, status, message)
```

Comment:
We need to handle the use case when no report is found in the database:
```suggestion
            report = session.query(Report).get(report_id)
            if report:
                res = self._setReviewStatus(
                    session, report.bug_id, status, message)
            else:
                raise codechecker_api_shared.ttypes.RequestFailed(
                    codechecker_api_shared.ttypes.ErrorCode.DATABASE,
                    "No report found in the database.")
```